client/.eslintrc 2.js

```diff
@@ -0,0 +1,173 @@
+module.exports = {
+    "env": {
```

Comment:
Why are we creating an eslintrc2.js?

Reply:
@johnagl Just removed this file. I think it was becasue I didnt pull before I started working on the branch.